mmv1/third_party/terraform/website/docs/r/compute_instance_template.html.markdown

```diff
@@ -241,6 +241,10 @@ The following arguments are supported:
     this template. This can be specified multiple times for multiple networks.
     Structure is documented below.
 
+* `network_performance_config` (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html)
+    Configures network performance settings for this template. Structure is
```

Comment:
Should the note about `GVNIC` apply to the instance template docs as well?

Reply:
Yes I can add that. I was thinking `compute_instance`'s documentation was considered the full docs source where things get verbose for this family of resources but there's little cost to having the same text in both places.